OrbitGl/DataViewTypes.h

```diff
@@ -15,6 +15,7 @@ enum class DataViewType {
   kSampling,
   kPresets,
   kAll,
+  kTracepoints
```

Comment:
The code e.g. OrbitQt\orbitmainwindow.cpp line 273 relies on "kAll" being the last entry. Move kTracepoint before kAll.